tests/sentry/runner/commands/test_configoptions.py

```diff
@@ -170,29 +171,40 @@ def test_sync(self):
             "sync",
         )
         assert rv.exit_code == 0, rv.output
-        output_before_log = "\n".join(
+        set_output = "\n".join(
             [
-                SET_MSG % ("int_option", 40),
-                UPDATE_MSG % ("str_option", "old value", "new value"),
-                SET_MSG % ("map_option", {"a": 1, "b": 2}),
-                SET_MSG % ("list_option", [1, 2]),
-                DRIFT_MSG % "drifted_option",
+                ConsolePresenter.SET_MSG % ("int_option", 40),
+                ConsolePresenter.SET_MSG % ("map_option", {"a": 1, "b": 2}),
+                ConsolePresenter.SET_MSG % ("list_option", [1, 2]),
             ]
         )
-        output_after_log = "\n".join(
+        drift_output = "\n".join(
             [
-                DB_VALUE % "drifted_option",
+                ConsolePresenter.DRIFT_MSG % "drifted_option",
+                ConsolePresenter.DB_VALUE % "drifted_option",
                 "- 1",
                 "- 2",
                 "- 3",
                 "",
-                CHANNEL_UPDATE_MSG % "change_channel_option",
-                UNSET_MSG % "to_unset_option",
```

Comment:
This message is not checked in the test anymore.

Reply:
An I moved it to lines 192-202, but in regards to your comment about checking the messages in sequence, I can just combine it all into one big ```expected_output```.

Reply:
Hold on. The UNSET message is not there anymore. I cannot see it in the new revision in any place.